src/context/SettingsContext.jsx

```diff
@@ -4,6 +4,7 @@ import { BTC } from '../utils'
 const localStorageKey = 'jm-settings'
 
 const initialSettings = {
+  theme: 'light',
   showBalance: false,
   unit: BTC,
```

Comment:
```suggestion
  unit: SATS,
```
Since these default values are already touched.. what do you think about making sats the standard?

Reply:
I feel like for JoinMarket most people use larger amounts (in the millions of sats) since super small amounts don't really make sense because of relatively high fees (compared to a lightning wallet where fees are in the single sat range at most). The [Bitcoin design guidelines](https://bitcoin.design/guide/payments/units-and-symbols/#recommended-interaction) say something along the lines of "sats for daily spending and BTC for savings applications". I guess JM is somewhere in between. I'd keep BTC the standard but definitely won't die on that hill. 😄

Reply:
I can understand this reasoning. Thanks for the explanation (and the link!). :muscle: